implementation/src/main/java/io/smallrye/config/DefaultValuesConfigSource.java

```diff
@@ -1,30 +0,0 @@
-package io.smallrye.config;
-
-import java.util.Collections;
-import java.util.Map;
-
-import org.eclipse.microprofile.config.spi.ConfigSource;
-
-public final class DefaultValuesConfigSource implements ConfigSource {
```

Comment:
Disagree: default values have to be represented by patterns thus `KeyMap`, and the set of properties it defines must be empty.

Reply:
It's ok. Let me bridge this with the API I've added in `SmallRyeConfigBuilder`.